Methods.cs

```diff
@@ -0,0 +1,231 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Numerics;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;

```

Comment:
I remember that we have talked about using other language than c++ but i do not recall that i allowed to use c#....

Reply:
please rewrite it in c++ (logic and code is almost the same so it is not too much work to do). Otherwise (i remember clearly myself saying this) - you are totally responsible for all aspects of programming (i do check only the final version). Currently you have plenty of style/code issues which will result probably in score close to 0 points.

Reply:
If this is you final version then probably you'll obtain 0pts (i read a few lines and already i can see -5 pts - for style)